### DISCUSSION TITLE
How to list groups with several meetings?

### DISCUSSION THREAD

**Discussion by kivisola** [code4recovery/12-step-meeting-list#599] [Q&A] [answered]
Hi!

What would be the proper way of listing several meetings held by the same AA group?

In Finland we don't list meetings per se, but groups. One group might have several meetings on different days.

How should these be entered to the system?

For example: Group Number One has two meetings, one on Mondays at 17 o'clock and another on Saturdays at 16 o'clock.

I guess they should be are separate entries to database. How to tell the db that these meetings have the same name, ie. the same group arranges these both meetings?

One solution would be adding the weekday after the name: "Group Number One Monday" and "Group Number One Saturday". But what if they change the meeting day from Saturday to Sunday? The name should be changed as well, also perhaps the URL too, and I think it would be confusing.

I have just come across this plugin and it seems wonderful. I have only superficial understanding of it, so maybe there is already a solution for this kind of situation.

**Comment by tech2serve** [comment, marked as answer]
Hello!

TSML expects each meeting to be entered separately, but you absolutely can then associate them with a Group. 

Scroll towards the bottom of the edit meeting page and you'll see a block titled Contact Information. Select "Part of a group" and then enter the group name. After you complete and publish that meeting, move onto the next meeting for that Group. As you enter the second meeting, simply add the same group name for it, and you should the other information from the group populate.

It should be that easy!

If you have troubles, let us know and we can provide screen caps.

**Reply by kivisola** [reply]
Okay, thank you!
